src/Integrations/ActiveCampaign/SettingsActiveCampaign.php

```diff
@@ -0,0 +1,383 @@
+	 *
+	 * @return array<int, array<string, mixed>>
+	 */
+	public function getSettingsGlobalData(): array
+	{
+		$isUsed = $this->isCheckboxOptionChecked(self::SETTINGS_ACTIVE_CAMPAIGN_USE_KEY, self::SETTINGS_ACTIVE_CAMPAIGN_USE_KEY);
+
+		$output = [
+			[
+				'component' => 'intro',
+				'introTitle' => \__('ActiveCampaign', 'eightshift-forms'),
+			],
+			[
+				'component' => 'intro',
+				'introTitle' => \__('How to get the API key?', 'eightshift-forms'),
+				'introTitleSize' => 'small',
+				// phpcs:ignore WordPress.WP.I18n.NoHtmlWrappedStrings
+				'introSubtitle' => \__('<ol>
+						<li>Log in to your ActiveCampaign Account.</li>
+						<li>Navigate to your user profile image (bottom left corner).</li>
+						<li>Click on <strong>Account</strong>.</li>
+						<li>Click on <strong>Extras</strong> and <strong>API Keys</strong> in the tabs section.</li>
+						<li>Click on the <strong>Create a Key</strong> button.<br/></li>
+						<li>Copy the API key into the field below or use the global constant.</li>
+					</ol>', 'eightshift-forms'),
+			],
+			[
+				'component' => 'divider',
+			],
+			[
+				'component' => 'checkboxes',
+				'checkboxesFieldLabel' => '',
+				'checkboxesName' => $this->getSettingsName(self::SETTINGS_ACTIVE_CAMPAIGN_USE_KEY),
+				'checkboxesId' => $this->getSettingsName(self::SETTINGS_ACTIVE_CAMPAIGN_USE_KEY),
+				'checkboxesIsRequired' => true,
+				'checkboxesContent' => [
+					[
+						'component' => 'checkbox',
+						'checkboxLabel' => \__('Use ActiveCampaign', 'eightshift-forms'),
+						'checkboxIsChecked' => $this->isCheckboxOptionChecked(self::SETTINGS_ACTIVE_CAMPAIGN_USE_KEY, self::SETTINGS_ACTIVE_CAMPAIGN_USE_KEY),
+						'checkboxValue' => self::SETTINGS_ACTIVE_CAMPAIGN_USE_KEY,
+						'checkboxSingleSubmit' => true,
+					]
+				]
+			],
+		];
+
+		if ($isUsed) {
+			$apiKey = Variables::getApiKeyActiveCampaign();
+			$apiUrl = Variables::getApiUrlActiveCampaign();
+
+			$output = \array_merge(
+				$output,
+				[
+					[
+						'component' => 'input',
+						'inputName' => $this->getSettingsName(self::SETTINGS_ACTIVE_CAMPAIGN_API_URL_KEY),
+						'inputId' => $this->getSettingsName(self::SETTINGS_ACTIVE_CAMPAIGN_API_URL_KEY),
+						'inputFieldLabel' => \__('API url', 'eightshift-forms'),
+						'inputFieldHelp' => \__('Can also be provided via a global variable.', 'eightshift-forms'),
+						'inputType' => 'text',
+						'inputIsRequired' => true,
+						'inputValue' => !empty($apiUrl) ? $apiUrl : $this->getOptionValue(self::SETTINGS_ACTIVE_CAMPAIGN_API_URL_KEY),
+						'inputIsDisabled' => !empty($apiUrl),
+					],
+					[
+						'component' => 'input',
+						'inputName' => $this->getSettingsName(self::SETTINGS_ACTIVE_CAMPAIGN_API_KEY_KEY),
+						'inputId' => $this->getSettingsName(self::SETTINGS_ACTIVE_CAMPAIGN_API_KEY_KEY),
+						'inputFieldLabel' => \__('API key', 'eightshift-forms'),
+						'inputFieldHelp' => \__('Can also be provided via a global variable.', 'eightshift-forms'),
```

Comment:
```suggestion
						'inputFieldHelp' => \__('Can also be provided via a global constant.', 'eightshift-forms'),
```

Reply:
this need to be changed on all integrations and multiple places so I will create an ticket tnx